app/models/anonymous_user.rb

```diff
@@ -9,6 +9,10 @@ def is_administrator?
     false
   end
 
+  def student?
```

Comment:
should this be `is_student` like the other methods? (just for consistency sake)

Reply:
No because note that this model is just for "anonymous" users — meaning that they are not logged in. And the `student?` method is not explicitly defined in the actual `User` model (logged in users)... Rails defines it for you with Ruby metaprogramming, so this `student?` method in the anon user just follows that convention, unlike the rest of the methods which Ruby does not define with metaprogramming `is_.*?` — those are different.